CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Enabled packaging the command line arguments when building a mobile client by default.
 
 ### Bug fixes:
+- Fixed a bug that was stopping Spatial from attempting to use travel URL (that was passed to `Browse`) if it couldn't connect via command line arguments.
```

Comment:
Fixed a bug that stopped the travel url being used for initial Spatial connection if the command line arguments could not be used.

Reply:
Done!